packages/access-client/src/agent-data.js

```diff
@@ -65,7 +80,7 @@ export class AgentData {
         delegation: importDAG(
           value.delegation.map((d) => ({
             cid: CID.parse(d.cid).toV1(),
-            bytes: d.bytes,
+            bytes: new Uint8Array(d.bytes),
```

Comment:
Avoid copy if already a `Uint8Array`?

```suggestion
            bytes: d.bytes instanceof Uint8Array ? d.bytes : new Uint8Array(d.bytes),
```